ckeditor-build/src/ckeditor.js

```diff
@@ -1,46 +1,51 @@
 /**
- * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
  * For licensing, see LICENSE.md.
  */
 
-import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
-import EssentialsPlugin from '@ckeditor/ckeditor5-essentials/src/essentials';
 import AutoformatPlugin from '@ckeditor/ckeditor5-autoformat/src/autoformat';
+import BlockquotePlugin from '@ckeditor/ckeditor5-block-quote/src/blockquote';
 import BoldPlugin from '@ckeditor/ckeditor5-basic-styles/src/bold';
-import ItalicPlugin from '@ckeditor/ckeditor5-basic-styles/src/italic';
-import StrikePlugin from '@ckeditor/ckeditor5-basic-styles/src/strikethrough';
+import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
 import CodePlugin from '@ckeditor/ckeditor5-basic-styles/src/code';
-import UnderlinePlugin from '@ckeditor/ckeditor5-basic-styles/src/underline';
+import EssentialsPlugin from '@ckeditor/ckeditor5-essentials/src/essentials';
 import HeadingPlugin from '@ckeditor/ckeditor5-heading/src/heading';
+import ItalicPlugin from '@ckeditor/ckeditor5-basic-styles/src/italic';
 import ListPlugin from '@ckeditor/ckeditor5-list/src/list';
 import ParagraphPlugin from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 import Range from '@ckeditor/ckeditor5-engine/src/model/range';
+import StrikePlugin from '@ckeditor/ckeditor5-basic-styles/src/strikethrough';
+import UnderlinePlugin from '@ckeditor/ckeditor5-basic-styles/src/underline';
 
 export default class ClassicEditor extends ClassicEditorBase {}
 
 ClassicEditor.build = {
   plugins: [
-    EssentialsPlugin,
     AutoformatPlugin,
+    BlockquotePlugin,
     BoldPlugin,
-    ItalicPlugin,
-    StrikePlugin,
     CodePlugin,
-    UnderlinePlugin,
+    EssentialsPlugin,
     HeadingPlugin,
+    ItalicPlugin,
     ListPlugin,
-    ParagraphPlugin
+    ParagraphPlugin,
+    StrikePlugin,
+    UnderlinePlugin,
   ],
   config: {
-    toolbar: [
-      'headings',
-      'bulletedList',
-      'numberedList',
-      'bold',
-      'italic',
-      'strike',
-      'code'
-    ]
+    toolbar: {
+      items: [
+        'headings',
+        'bulletedList',
+        'numberedList',
+        'bold',
+        'italic',
+        'strike',
```

Comment:
It won't work. The correct name is `strikethrough`. See https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/basic-styles.html.

Reply:
So we have the config here: https://github.com/mozilla/notes/blob/master/src/sidebar/app/data/editorConfig.js#L14 

I always get confused why we need this array here anyway, because we initialize with a different config.  Are these the defaults or something?

Reply:
This is default config for the build. If you won't specify the toolbar items during the editor initialization (`ClassicEditor.create( element )`), the editor will use the built-in config.

I guess, in your case the `ClassicEditor.build` property could be skipped.